### DISCUSSION TITLE
Satellite Image Thumbnail

### DISCUSSION THREAD

**Discussion by bab014** [weather-gov/api#263] [Q&A]
I am trying to understand the /thumbnails/satellite/{area} API call and I can't find any information on what are valid areas.

Is there something I am missing in the documentation?

**Comment by StephenClouse** [comment]
This endpoint is deprecated and will be removed in a future release.

We would advise not using it, as it almost certainly does not do what you probably think it does. This is part of why we're getting rid of it.

**Comment by bab014** [comment]
Ah, good to know.

Is there a way to get a radar/satellite image from the API?

**Comment by StephenClouse** [comment]
Satellite imagery can be found at [NESDIS](https://nesdis.noaa.gov).
